crates/extensions/tedge_health_ext/src/lib.rs

```diff
@@ -39,8 +40,10 @@ impl HealthMonitorBuilder {
         mqtt: &mut (impl ServiceProvider<MqttMessage, MqttMessage, TopicFilter> + AsMut<MqttConfig>),
         // TODO: pass it less annoying way
         mqtt_schema: &MqttSchema,
-        mut service_type: String,
+        service_config: &TEdgeConfigReaderService,
```

Comment:
I also modified this to take the relevant chunk of the config rather than two separate arguments for the type and time format, which means a couple of tests now use the new method `TEdgeConfigRepository::from_toml_str` to generate the relevant config.